piet/src/gradient.rs

```diff
@@ -61,7 +61,7 @@ pub struct FixedRadialGradient {
 }
 
 /// Specification of a gradient stop.
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
Minor nit, but order is not consistent. I'm not sure what "best practice" is here and it doesn't really matter, but for types nearby in the source it would be nice to be consistent.